htdocs/core/class/commonobject.class.php

```diff
@@ -4298,11 +4298,12 @@ public function fetchObjectLinked($sourceid = null, $sourcetype = '', $targetid
 			}
 		}
 		$sql .= " ORDER BY ".$orderby;
-
+		dol_syslog("sql=".$sql, LOG_DEBUG);
```

Comment:
There is no need to add a dol_sylog here. Any execute sql contains a dol_sylog before beeing executed.

Reply:
yeah, sorry, that is a leftover from my debugging, I'll remove it when I get my end result working.

I simply wish that when I have linked an invoice/(order/propal) to a subscription, then the link will show up not only in the subscription, but also in the invoice/(order/propal)